### ISSUE TITLE
nativescript

### ISSUE THREAD

**Issue by immae1** [mchl18/reactcup#1]
if like to use vue or angular maybe this is a nice solution: https://www.nativescript.org

**Comment by mchl18** [comment]
So one of the coolest things about react native is that it transpiles to js only.
This means we could e.g. reuse this svg renderer in another stack.
However, we are already reliant on `expo` so far for this mobile build but I gotta say I had it running super fast. This is perfect for prototyping components and elements.

I think, if we want to keep on with this we want to build it either in the cloud or locally.
I tend to stay with react native since it seems to have the best open source support.

Any thouhts? @immae1

**Comment by mchl18** [comment]
I also very much like the thought of JSX for rendering DOM instead of using seperate HTML. I believe it encourages cleaner code.

**Comment by immae1** [comment]
Allright then lets use this stack. i created a docker setup for you - but there are some small issues left. give me some time.

**Comment by mchl18** [comment]
https://market.nativescript.org/?tab=samples&framework=all_frameworks&category=all_samples

This is actually a nice collection of code samples :D

**Comment by mchl18** [comment]
Yeah no worries, I have spent a lot of time in the structure and setup but things seem to be working somewhat how I like them so I will continue rebuilding basic stuff but I think this is very doable!

**Comment by immae1** [comment]
@mchl18 if you pull the new changes you can use docker  ;)

**Comment by mchl18** [comment]
Wohoo you da man @immae1
